mcs/class/System.Data.OracleClient/Test/System.Data.OracleClient.jvm/OracleCommand/OracleCommand_Dispose.cs

```diff
@@ -83,8 +83,7 @@ public static void Main()
 	//bool bFound;
 	//try
 	//{
-	//Connect = new OracleConnection("Provider=\"MSDAORA.1\";User
-	//ID=COM;Password=SQL;Data Source=LLO1;HostName=WS10359;Port=1521");
+	//Connect = new OracleConnection("[PLACEHOLDER]");
```

Comment:
can we get rid of the entire commented block ?

Reply:
Yes, but I'm inclined to just do as little damage as possible since it wasn't obvious to me whether or not the block was useful. It's an email describing an issue that led to the test, so not necessarily a good candidate for deletion?